app/controllers/users_controller.rb

```diff
@@ -1,35 +1,36 @@
+# frozen_string_literal: true
+
 class UsersController < ApplicationController
-  helper_method :cause_logos ##will delete when we get real org logos.
-  before_action :get_user, :get_donations_by_causes, only: [:show, :edit]
+  helper_method :cause_logos # #will delete when we get real org logos.
+  before_action :user, :donations_by_causes, only: %i[show edit]
 
   def show
     @badges = @user.badges
   end
 
-  def edit 
+  def edit
     @user_fav_orgs = @user.user_favorite_organizations[0..3]
   end
 
-  def get_user
+  def user
     @user = User.includes(donations: :organization, user_favorite_organizations: :organization).find(params[:id])
   end
 
-  def get_donations_by_causes
+  def donations_by_causes
     @donations_by_causes = @user.donations_by_causes
   end
 
   def cause_logos
     {
-      "animals"        => "paw",
```

Comment:
I guess rubocop doesn't appreciate readability. @uuunk what's the rule against this?

Reply:
There's a migration where I do something similar, but that wasn't flagged. I don't quite understand what's going on here.

Reply:
He's extending off of the default config, which is here https://github.com/rubocop-hq/rubocop/blob/master/config/default.yml

Reply:
do you know the exact rule it's using to get rid of that space? I'm just wondering why some things are flagged, but others aren't.

Reply:
Hmm no I couldn't find anything that would keep that whitespace. Maybe missing. If we come across it I'd be happy to turn it off to make that kind of stuff more readable. I think it sees it as arbitrary whitespace.